packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.spec.js

```diff
@@ -0,0 +1,31 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+
+import TranscribedLines from './TranscribedLines'
+
+let wrapper
+
+describe('Component > TranscribedLines', function () {
+  before(function () {
+    wrapper = shallow(<TranscribedLines />)
+  })
+
+  it('should render without crashing', function () {
+    expect(wrapper).to.be.ok()
+  })
+
+  describe('incomplete lines', function () {
+    it('should be red', function () {})
+    it('should be focusable', function () {})
+    it('should be clickable', function () {})
+  })
+
+  describe('completed lines',function () {
+    it('should be grey', function () {})
+    it('should be labelled with the consensus text', function () {})
+    it('should show the label on hover', function () {})
+    it('should be focusable', {})
+    it('should show the label on focus', function ())
+    it('should not be clickable')
```

Comment:
The focus/hover state of the completed lines is a help tooltip text:

> As a user I want to hover over others’ previous markings to see a hover state explaining that these markings are other users’ markings.

The consensus text is something that is from the click event to open up the sub-task that is no longer editable (or something similar)

> As a user I want to be able to read completed (greyed-out) transcriptions. 

Is there a reason to not do it like this?